### PR TITLE
faq link dynamic

### PR DESCRIPTION
**Pull request by hamzanaveed9**
faq link dynamic

### CODE REVIEW

File: app/views/spree/shared/_footer.html.erb [hunk 1]
```diff
@@ -86,7 +86,107 @@
 <%# footer is replaced overall %>
 <% cache spree_menu_cache_key('footer') do %>
   <footer id="footer" class="footer-wrapper" >
-    <%= render partial: 'spree/shared/upper_footer' %>
```
Comment (omairazampunch) [inline app/views/spree/shared/_footer.html.erb]: @hamzanaveed9 , why we removed this partial? Any specific reason?

Comment (hamzanaveed9) [reply app/views/spree/shared/_footer.html.erb]: resolved it sir
just moved it here to make it dynamic

File: app/views/spree/cms_pages/show.html.erb [hunk 2]
```diff
@@ -1,16 +1,18 @@
-<% if @cms_page %>
```
Comment (om-ma) [inline app/views/spree/cms_pages/show.html.erb:1]: why we remove this check

File: app/views/spree/cms_pages/show.html.erb [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
-  <% if simple_page?(@cms_page) %>
```
Comment (om-ma) [inline app/views/spree/cms_pages/show.html.erb:2]: why we remove this check

File: app/views/spree/cms_pages/show.html.erb [hunk 4]
```diff
@@ -1,16 +1,18 @@
...
-  <% else %>
-      <% @cms_page.cms_sections.each do |section| %>
-        <%= build_section(section) %>
-      <% end %>
-  <% end %>
-<% end %>
-<%= render "spree/shared/cms/pages/edit_mode", edit_mode: @edit_mode %>
+  </div> -->
+  <%# else %>
+      <%# @cms_page.cms_sections.each do |section| %>
```
Comment (om-ma) [inline app/views/spree/cms_pages/show.html.erb:11]: also add else case in it

**Pull request merged**